test/core/public/run-rules.js

```diff
@@ -344,7 +344,7 @@ describe('runRules', function() {
 		});
 
 		fixture.innerHTML =
-			'<div id="t1"><span></span></div><div id="t2"><em></em></div>';
+			'<div id="t1"><em></em></div><div id="t2"><em></em></div>';
```

Comment:
We no longer have karma output in the test file so these selectors were now unique and the target didn't need a child selector. Updated to force non-unique nodes